README.md

```diff
@@ -22,7 +22,8 @@ To install `zq` or any other tool from this repo, you can either clone the repo
  [release](https://github.com/brimsec/zq/releases), available for Windows, macOS, and Linux.
 
 If you don't have Go installed, download and install it from the
-[Go downloads page](https://golang.org/dl/).
+[Go downloads page](https://golang.org/dl/). Go version 1.14 or later is
+required.
```

Comment:
I'm not sure if we should say "1.14 or later" or just "1.14" (and then change update to 1.15 when that happens, etc). @alfred-landrum ?

Reply:
I'm good with "1.14 or later".

Reply:
Cool. Yeah, i was just mimicking what I saw in CONTRIBUTING. :)